### DISCUSSION TITLE
Different GOS in different versions

### DISCUSSION THREAD

**Discussion by tongzhangqu** [mtex-toolbox/mtex#2006] [Ask Anything]
Hi there, 

I found grains.GOS in version 5.10.0 is the half of GOS in version 5.8.2. I assume the computation has been somehow changed?  

Best wishes
Tongzhang

**Comment by kilir** [comment]
Yes, that is correct. Introduced here 4daa022.
The reason is that using acumarray is much faster. You can switch back to the old behavior in thise section if you like.

Cheers,
Rüdiger
